app/templates/gulpfile.js

```diff
@@ -7,11 +7,13 @@ const wiredep = require('wiredep').stream;
 const runSequence = require('run-sequence');
 const autoprefixer = require('autoprefixer');
 const cssnano = require('cssnano');
+const argv = require('yargs').argv;
```

Comment:
So sorry to nitpick, but you can destructure this instead:
```suggestion
const { argv } = require('yargs');
```

Reply:
Right! Let me change that 👍